src/styles/events.scss

```diff
@@ -274,50 +294,57 @@
 		margin-top: 2px;
 		display: flex;
 		justify-content: space-between;
-		align-items: flex-start;
+		align-items: center;
 		font-size: 10px;
 	}
 
 	&__id {
 		color: $eventCardIdColor;
+		flex-shrink: 2;
+		overflow: hidden;
+		white-space: nowrap;
+		text-overflow: ellipsis;
 	}
 
 	&__timestamp {
 		justify-content: flex-end;
 		display: flex;
+		text-overflow: ellipsis;
 		flex-wrap: wrap;
 		min-width: 0;
 		margin-left: 5px;
+		padding: 0 2px;
+		transition: all 0.15s;
+		border: 1px solid transparent;
+		border-radius: 4px;
+		flex-shrink: 0;
+		white-space: nowrap;
+		overflow: hidden;
+
+		&:hover {
+			color: $graphTextColor;
+			border: 1px solid $graphTextColor;
+		}
 	}
 
 	&__timestamp-item {
+		text-align: right;
 		margin-left: 5px;
 		display: flex;
 		min-width: 0;
 	}
 
-	&__timestamp-label {
-		font-size: 9px;
-		color: $alternativeTextColor;
-	}
-
-	&__timestamp-value {
-		margin-left: 5px;
-		font-size: 10px;
-		color: $activeTextColor;
-		font-weight: 600;
-		overflow: hidden;
-		white-space: nowrap;
-		text-overflow: ellipsis;
-	}
-
 	&__bookmark {
 		display: flex;
 		align-items: center;
 		flex-shrink: 0;
 	}
 }
 
+.closed {
+	display: none;
+}
```

Comment:
this should be inside of an element you close. Currently it's a global rule, that may give problems in future